ioctl/cmd/action/actioninvoke.go

```diff
@@ -28,9 +28,10 @@ var (
 
 // actionInvokeCmd represents the action invoke command
 var actionInvokeCmd = &cobra.Command{
```

Comment:
pls add a comment here - 
"notify users to use the new ioctl contract command; 
TODO: this command will be deprecated soon"

same above